modules/serverless-rn-1-13-0.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-13-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} 1.13.0
+
+[id="new-features-1-13-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.19.0.
+* {ServerlessProductName} uses Knative Eventing 0.19.2.
+* {ServerlessProductName} now uses Kourier 0.19.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.19.1.
+* {ServerlessProductName} now uses Knative Kafka 0.19.1.
+* A `DomainMapping` custom resource has been added to {ServerlessProductName} to enable users to map a custom domain name to a Knative Service. For more information, see the https://knative.dev/docs/serving/creating-domain-mappings/[Knative documentation].
+
+[id="fixed-issues-1-13-0_{context}"]
+== Fixed issues
+
```

Comment:
Would like to mention this issue in the "Fixed issues" section: https://issues.redhat.com/browse/SRVKS-641

The rest of the release notes looks good to me! Thanks

Reply:
@mgencur this one was never included in the 1.10.0 release notes presumably because it's an upstream issue that was fixed upstream. I don't think it needs to be included here since we never reported it downstream as a known issue. LMK if you have any major objections.